src/database/queries.ts

```diff
@@ -297,21 +297,21 @@ export const getVouchersByVendorUuid = async (
 };
 
 /**
- * Query the `transactions` collection and return a Transaction if the uuid is found.
+ * Query the `invoices` collection and return an Invoice if the uuid is found.
  */
-export const getTransaction = async (uuid: Uuid): Promise<Transaction> => {
+export const getInvoice = async (uuid: Uuid): Promise<Invoice> => {
   try {
-    const dbQuery = query(transactionCollection, where('uuid', '==', uuid));
+    const dbQuery = query(invoiceCollection, where('uuid', '==', uuid));
     const querySnapshot = await getDocs(dbQuery);
-    return querySnapshot.docs[0]?.data() as Transaction;
+    return querySnapshot.docs[0]?.data() as Invoice;
   } catch (e) {
     // eslint-disable-next-line no-console
-    console.warn('(getTransaction)', e);
+    console.warn('(getInvoice)', e);
     throw e;
```

Comment:
Is there a cap on the number of vouchers able to be added to an invoice? I noticed that the $5100 invoice only had 51 vouchers. Was the value of the invoice manually inputted in the database, or was there some sort of error here?
<img width="515" alt="Screen Shot 2023-05-15 at 3 35 32 PM" src="https://github.com/calblueprint/vouchers4veggies/assets/66931067/2662b034-047c-420d-bda8-79291ff7a827">